src/services/pin/pin-model.js

```diff
@@ -100,8 +100,10 @@ pinSchema.pre('findOne', function populateFields(next) {
 });
 
 autoIncrement.initialize(mongoose);
-pinSchema.plugin(autoIncrement.plugin, { model: 'Pin', field: 'issue_id' });
+pinSchema.plugin(autoIncrement.plugin, { model: 'Pin', field: 'issue_id', startAt: 1703 });
 
 const Pin = mongoose.model('Pin', pinSchema);
 
+Pin.resetCount((err, nextCount) => {});
```

Comment:
What is this for? Does it reset counter when node restarts?

Reply:
Just for once. After run, I will remove and PR without it again